packages/vite/src/node/ssr/fetchModule.ts

```diff
@@ -46,19 +46,17 @@ export async function fetchModule(
 
     const resolved = tryNodeResolve(url, importer, {
       mainFields: ['main'],
-      conditions: [],
+      conditions: [...externalConditions, 'production', 'development'],
```

Comment:
Shouldn't these two also move to default config constants similar to `conditions`?

Reply:
`production` and `development` should not be included in `externalConditions` as they are not set by default in Node. So I removed `production` and `development` instead.

Reply:
Yeah, I remembered that was mentioned during the meeting. That sounds good to me too :+1: